egs/babel_multilang/s5/run-2c-bnf.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# -v3 is as -v2 but it just consists 10 closest fullLP langs to GRG + GRG.
```

Comment:
i think these comments may be out of date.  Comments at the top of this script should say what the script does... not clear to me right now.